docs/en/api-reference/ngsild.md

```
@@ -404,4 +417,1186 @@ DELETE /ngsi-ld/v1/entities/{entityId}/attrs/{attrName}?deleteAll=true
+      {
+        "@value": { "avg": 21.0 },
+        "observedAt": "2024-01-01T10:00:00Z",
+        "endAt": "2024-01-01T11:00:00Z"
+      }
+    ]
+  }
+}
+```
+
+> **Note**: Specifying `aggrMethods` without `aggrPeriodDuration` returns a `400 Bad Request` error.
+
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)